.github/workflows/ci.yml

```diff
@@ -19,14 +19,14 @@ on:
 
 jobs:
   ci-does-not-run-on-draft-pull-requests:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Why are we making this change?

Reply:
updating description now

Reply:
Great question. It seems to be the cause of why some PRs get stuck in the "waiting for status to be reported" state (which I've been seeing with #8499). 

Other users have been reporting similar problems (which I've linked in the PR description), and for us at least, it appears that the `ubuntu-23.04` is causing the problem for us.

Reply:
trying again with `23.04` to make sure

Reply:
Yup, it seems to work with `22.04` only